src/main/java/org/springframework/data/couchbase/core/ReactiveFindByQueryOperationSupport.java

```diff
@@ -70,7 +70,7 @@ public ReactiveFindByQueryOperationSupport(final ReactiveCouchbaseTemplate templ
 
 		@Override
 		public Mono<T> one() {
-			return all().single();
+			return all().singleOrEmpty();
```

Comment:
If this is changed here, does it also need to be changed for analytics?

Reply:
Yes, it probably needs to be changed there as well. I'll take a look.  I'll look for 'single()' everywhere.